watson-discovery/watson-discovery.lua

```diff
@@ -0,0 +1,94 @@
+--[[
+# [Watson API Explorer]https://console.bluemix.net/docs/services/discovery/getting-started-tool.html#getting-started-with-the-tooling) connector.
+
+[Discovery](https://watson-api-explorer.mybluemix.net/apis/discovery-v1#!/Configurations/listConfigurations)
+
+## Input parameters
+* watson_domain - e.g. `https://watson-api-explorer.mybluemix.net/`.
+* api_endpoint - API endpoint URI, e.g. `personality-insights/api/v3/profile`.
+* text - A maximum of 20 MB of content to analyze, though the service requires much less text.
+
```

Comment:
there is no `version`, `username`, `password` parameters which are required according to your implementation